src/lib.rs

```
@@ -1,4 +1,4 @@
-use std::fmt::format;
+use std::{cmp::Ordering, fmt::format};
```

Clippy (1.93.0)

[warning] unused import

(warning)